src/App.jsx

```diff
@@ -1,44 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, onTableSelected] = useState('');
```

Comment:
The state updater function `onTableSelected` should be renamed to `setActiveTabId` to follow the convention of naming state updater functions with a `set` prefix. This will also help avoid confusion, as the current name suggests it might be related to a table instead of a tab.